### PR TITLE
Bump CUQUANTUM libraries

### PR DESCRIPTION
**Pull request by maleadt**

### CODE REVIEW

**Review by kshyatt — APPROVED** [review]

**Comment by maleadt** [comment]
No API adjustments needed?

**Comment by kshyatt** [comment]
I was going to work on those in a separate PR but can push to this one if you'd prefer.

**Comment by maleadt** [comment]
Rebased, hopefully that gets rid of the CUSPARSE failure.

**Comment by maleadt** [comment]
Ah, there are the relevant CI failures 😅

**Comment by kshyatt** [comment]
Now including wrappers for new CUTENSORNET functions.

File: lib/cutensornet/src/libcutensornet.jl [hunk 1]
```diff
@@ -669,11 +669,11 @@ end
     initialize_context()
     @ccall libcutensornet.cutensornetTensorSVD(handle::cutensornetHandle_t,
                                                descTensorIn::cutensornetTensorDescriptor_t,
-                                               rawDataIn::Ptr{Cvoid},
+                                               rawDataIn::CuPtr{Cvoid},
```
Comment (maleadt) [inline lib/cutensornet/src/libcutensornet.jl:672]: corresponding changes in the `wrap`?

**Pull request merged**